packages/auth/src/authClients/firebase.ts

```diff
@@ -14,6 +14,14 @@ export type oAuthProvider =
   | 'microsoft.com'
   | 'apple.com'
 
+export type PasswordProvider = { email: string; password: string }
+
+const isPasswordProvider = (
+  usingProvider: oAuthProvider | PasswordProvider
+): usingProvider is PasswordProvider => {
+  return (usingProvider as PasswordProvider).email !== undefined
```

Comment:
How about checking for password too?
```suggestion
const provider = usingProvider as PasswordProvider
  return provider.email !== undefined || provider.password !== undefined
```

I know someone was looking at email link authentication too

Reply:
I added a check that email and password are both not undefined, this will allow for a future email sign in type.